Header header
int32 rows
int32 cols
int32 robot_x_idx
int32 robot_y_idx
int32[] cells